src/utils/date.ts

```diff
@@ -1,26 +1,25 @@
 import { DateTime } from 'luxon';
 
-export const parseAsDate = (date: string) =>
-  DateTime.fromISO(date, { zone: 'utc' });
+// Used to parse out mintute, day, ,month, and years from ISOString
+export const parseAsUTC = (date: string) => DateTime.fromISO(date).toUTC();
 
-export const parseAsLocalTime = (date: string) => DateTime.fromISO(date);
+type DateFormat = 'MM/dd/yyyy' | 'MMMM d, yyyy';
```

Comment:
Should a regular `string` also be part of this literal union? I understand that the restriction narrows down all the variation we ended up with, but it makes these helpers less generic. The type would need an update every time there is a new format, but maybe that's the intent?

Adding string to the union would let these functions match the argument for `toFormat`, and would also provide some code hints about the common options.

Reply:
Also, did you decide that there shouldn't be a default format anymore? I think choosing one is okay.

Reply:
On second thought DateFormat is okay as it is, since the point is to narrow down date formatting options.

Reply:
I think the point of the union is to define the types of date formats that EASI uses.  I can see with the addition of `string`, the formats might start to get varied and not EASI standard.  At the end of the day it's not a huge deal, I would okay with it, that was just my line of thinking